Readme.md

```diff
@@ -31,10 +31,14 @@ Here's an example of what an app shows up as on the Plex web interface
 
 The rows in that example from top to bottom are `deviceName`, `version`, `product`, and `device`.
 
-### .query(uri)
+### .query(parameters)
 
 **Retrieve content from URI**
 
+The parameter can be a string representing the URI, or an object with the following properties (everything is optional after the uri):
+- **uri**: the URI to query
+- **extraHeaders**: an object with extra headers to send in the HTTP request. Useful for things like X-Plex-Target-Client-Identifier
```

Comment:
Prepending `(optional)` to this extraHeaders description, would be more explicit than the last part of line #38.